docs/testnet.md

```diff
@@ -15,14 +13,11 @@ The following is a break down of the intended testnet roadmap, the features that
 - [x] [Dev Net 1](#devnet-1)
 - [x] [Dev Net 2](#devnet-2)
 - [x] [Athena Public Testnet 1](#athena-public-testnet-1)
-- [ ] [Bia Attack net](#bia-attack-net)
-- [ ] [Circe Public Testnet 2](#cerce-public-testnet-ii)
-- [ ] [Demeter Red/Blue net](#demeter-redblue-net)
+- [ ] [Bia Public Testnet 2](#bia-public-testnet-2)
+- [ ] [Circe Attack Net](#cerce-attack-net)
```

Comment:
Looks like I've been mispelling it and it's Circe not Cerce 😬  https://en.wikipedia.org/wiki/Circe#:~:text=Circe%20(%2F%CB%88s%C9%9C%CB%90rs,knowledge%20of%20potions%20and%20herbs.